sharktank/sharktank/ops/signatures.py

```diff
@@ -125,16 +125,27 @@ def _all_reduce_trampoline(d: SignatureDispatcher, tensor: AnyTensor):
 
 
 @overridable
-def argmax(tensor: AnyTensor, axis: int) -> AnyTensor:
+def argmax(
+    tensor: AnyTensor,
+    dim: Optional[int] = None,
+    chunk_size: Optional[int] = None,
```

Comment:
You will need to update the `sharded_impls.py` version as well so that it has `chunk_size`. It should just involve proxy-ing the value to the sub calls.